android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java

```diff
@@ -586,4 +622,28 @@ private ViewManager resolveViewManager(int tag) {
       return null;
     }
   }
+
+  private Point convertAbsoluteToParentRelative(Point fromPoint, View parentView) {
```

Comment:
convertScreenLocationToViewCoordinates

Reply:
Resolved in: https://github.com/software-mansion/react-native-reanimated/pull/4014/commits/e65b1e008ca12fcb14d9fce05e3ff30fe24381cb